bot/exts/evergreen/minesweeper.py

```diff
@@ -78,7 +79,7 @@ class Game:
 class Minesweeper(commands.Cog):
     """Play a game of Minesweeper."""
 
-    def __init__(self, bot: commands.Bot) -> None:
+    def __init__(self, _bot: Bot) -> None:
```

Comment:
Why do you still pass a bot reference here? It isn’t used anywhere afaik.

Reply:
Resolved in https://github.com/python-discord/sir-lancebot/pull/718/commits/c051abf49ce5bb82d127c8ddfad7ece431bb028f.